src/drag-drop-context.tsx

```diff
@@ -52,10 +53,10 @@ interface Sensor {
   activators: { [K in keyof HTMLElementEventMap]?: SensorActivator<K> };
 }
 
-type Transformer = (
+type Transformer = ((
   transform: Transform,
   context: { type: "draggables" | "droppables"; id: string | number }
-) => Transform;
+) => Transform) & { draggableId: string | number };
```

Comment:
`draggableId` here is a bit of an internal hack so should make optional for clarity.